dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/ServiceRegistryService.java

```diff
@@ -15,6 +17,16 @@
     private final URI uri;
     private final List<ServiceAcl> acls;
 
+    public static ServiceRegistryService of(URI uri, List<com.lightbend.lagom.javadsl.api.ServiceAcl> acls) {
+        List<ServiceAcl> internalAcls =
+            acls.stream().map(api -> {
```

Comment:
Nitpicking:

`acls.stream().map(api -> { ... })` to `acls.stream().map(acl -> { ... })`